### PR TITLE
Basic server setup + tests

### PR DESCRIPTION
**Pull request by ShaneSutro**
@jsmithb117  - would you mind reviewing the below when you get a chance? Thank you!

 - [X] Added server route for /api/about/:id
 - [X] Added basic react component to render to page and confirm webpack
 - [X] Added model.js file to interact with database from component/server
 - [X] Added hotloader for webpack during development
 - [X] Added tests for server routes

### CODE REVIEW

File: database/model.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const { descriptionSchema } = require('./db');
+
+const Description = mongoose.model('Description', descriptionSchema);
+
+module.exports = {
+  getOne: (id) => (
```
Comment (jsmithb117) [inline database/model.js:7]: I never thought to add the database functions as methods of exports.  This is pretty genius!

File: public/index.html [hunk 2]
```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script src="https://use.fontawesome.com/cb5b364a9a.js"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap" rel="stylesheet">
```
Comment (jsmithb117) [inline public/index.html:5]: I've been thinking about what font to use.  I'm gonna steal this!

**Review by jsmithb117 — APPROVED** [review]
Looks good!

File: public/index.html (hunk 2 above)
Comment (ShaneSutro) [reply public/index.html:5]: @jsmithb117 please do! From what I saw, this is the one they use on their site.

**Pull request merged**